Automattic/wp-calypso
# Move more selectors under state/reader.

This is a follow-up to #39261, where Reader state was modularized.

To keep that PR smaller, I opted to leave some of the selector moving to a subsequent PR (this one). 

#### Changes proposed in this Pull Request

* Move reader-related selectors under `state/reader`.
* Drive-by fixes of some minor lint issues.

#### Testing instructions

Since this PR doesn't deal with changes other than code reorganisation, it should be sufficient to ensure that there are no build errors or warnings about missing exports. Giving Reader a general spin would add even more confidence.
